src/graphics/basic_components/slider.ts

```diff
@@ -42,6 +42,7 @@ export class Slider {
       if (props.onChange) {
         props.onChange(value);
       }
+      this.sliderInput.blur();
```

Comment:
This fixes cases where the user moves the slider, but not the ones where the user just clicks on the slider. For that, we probably need to add the same on the onclick handler.